modules/javafx.base/src/main/java/javafx/event/Event.java

```diff
@@ -41,6 +41,7 @@
  * to events of the same {@code Event} class.
  * @since JavaFX 2.0
  */
+@SuppressWarnings("doclint")
 public class Event extends EventObject implements Cloneable {
```

Comment:
I don't think we should suppress all warnings, especially at the class level.

Reply:
for some reason I though it did not work -- corrected, fixed by suppressing the offending lines only with `@SuppressWarnings("doclint:missing")`